content/_index.de.md

```diff
@@ -33,19 +33,19 @@ Diese grundlegenden Werte und Prinzipien sind:
 
 * Wir haben die Verpflichtung unsere – wie subtil auch immer – privilegierten Positionen zu nutzen, um das Leben Anderer zu verbessern.
 
-* Wir müssen Raum schaffen für Menschen, die nicht so sind wie wir, damit sie in unseren Bereich einsteigen und darin erfolgreich sind. Das bedeutet nicht nur, diese Menschen in den IT-Bereich einzuladen, sondern auch sicherzustellen, dass sie darin Unterstützung und Ermächtigung finden.
+* Wir müssen Raum schaffen für Menschen, die nicht so sind wie wir, damit sie in unserer Branche einsteigen und erfolgreich sind. Das bedeutet nicht nur, diese Menschen in die IT-Branche einzuladen, sondern auch sicherzustellen, dass sie darin Unterstützung und Ermächtigung finden.
```

Comment:
I think ["einsteigen"](https://www.duden.de/rechtschreibung/einsteigen) is always combined with the accusative form of the object that you enter into. Then it would be

```
damit sie in unsere Branche einsteigen und [...]
```